### PR TITLE
27 feature hover badges

### PR DESCRIPTION
**Pull request by VitorMeirelesOliveira**
-Modified the badges.js file with the css style to put the tooltip

Signed-off-by: Vitor Meireles Oliveira <vitorrr.meireles@hotmail.com>
Co-authored-by: Francisco <heronildo.2010@hotmail.com>
Co-authored-by: Jaco Apolinario <jacoappolinario@hotmail.com>
Co-authored-by: cjjcastro <cjj.castro3@gmail.com>

## Description
Created tooltip for badges

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by FranciscoHeronildo** [comment]
The messages of the hovers of the badges were changed respecting the required criteria. Please reopen the PR and verify.

File: app/scripts/badges.js [hunk 1]
```diff
@@ -12,14 +12,14 @@ export default () => [
 
 
 '<div style="width: 100px; height: 100px; display: inline; padding-right: 5px" id="my-badge" class="tooltip">',
-'<span class="tooltiptext">Percentage of Repository Activity</span>',
+'<span class="tooltiptext">Percentage calculated based on the number of commits, the number of contributors, accepted PRs, and active issues of the last month. Also, if release notes exist.</span>',
```
Comment (filipetoyoshima) [inline app/scripts/badges.js]: About PRs, is not only acceptance that affects the total Score. Everything that can happen in PRs is considered.
The release note not only must exist but must be recent.

File: app/scripts/badges.js [hunk 2]
```diff
@@ -12,14 +12,14 @@ export default () => [
...
 '</div>',
 
 '<div style="width: 100px; height: 100px; display: inline; padding-right: 5px" id="my-badge2" class="tooltip">',
-'<span class="tooltiptext">Percentage of User Support</span>',
+'<span class="tooltiptext">Percentage calculated based on the checklist of the community profile, the number of contributors and accepted PRs and also based on tags, features, and labels for PRs and issues.</span>',
```
Comment (filipetoyoshima) [inline app/scripts/badges.js]: > and also based on tags, features, and labels for PRs and issues.

What are those tags, features, and labels?

File: app/scripts/badges.js [hunk 3]
```diff
@@ -12,14 +12,14 @@ export default () => [
...
 '</div>',
 
 '<div style="width: 100px; height: 100px; display: inline; "id="my-badge3" class="tooltip">',
-'<span class="tooltiptext">Percentage of Contribution Receptiveness</span>',
+'<span class="tooltiptext">Percentage calculated based on the checklist of the community profile, the existence of release note as well and the number of active issues.</span>',
```
Comment (filipetoyoshima) [inline app/scripts/badges.js]: Same as Activity about Release Notes. It must be recent, not only exist.

**Comment by filipetoyoshima** [comment]
Don't forget to reopen the PR when the changes are made.

File: app/scripts/badges.js (hunk 2 above)
Comment (VitorMeirelesOliveira) [reply app/scripts/badges.js]: labels as "good first issue" and "help wanted", for example.

**Pull request merged**